module/os/freebsd/zfs/zfs_znode.c

```diff
@@ -2034,6 +2034,20 @@ zfs_obj_to_stats(objset_t *osp, uint64_t obj, zfs_stat_t *sb,
 	return (error);
 }
 
+
+void
+zfs_inode_update(znode_t *zp)
```

Comment:
`zfs_znode_update` seems a more appropriate name now.

Reply:
I'm not sure it is, since the point of it is to update the underlying OS data structure _not_ the znode.